docs/office-hours/20190903_meeting_notes.md

```diff
@@ -0,0 +1,120 @@
+# osquery Office Hours 2019-09-03
+
+## Highlights since the last meeting
+
+* Lots of progress towards [4.0.1](https://github.com/osquery/osquery/milestone/42)
+  - (This has its own agenda item)
+  - Changelog merged (big thanks to @sharvilshah)
+  - Third party libraries merged (https://github.com/osquery/osquery/pull/5706)
+* [Views created on startup](https://github.com/osquery/osquery/pull/5732)
+* Several documentation updates. Build instructions, and Windows packaging.
+* Additional tests for tables
+* Remove FTS features from sqlite (in response to vulnerability) (See https://github.com/osquery/osquery/issues/5702 for background)
+
+
+## 4.0.1
+
+### Third Party Libs
+
+https://github.com/osquery/osquery/pull/5706 merged. This means third party libraries are now build from source! Huge work from @alessandrogario, @smjert, @theopolis
+
+### Toolchain
+Custom tool chain to build osquery is not yet merged. The toolchain is ready, and seems to work. You can build with it, and tests pass. But integrating it with the third party dependencies is surprisingly hard. This is partly due to the age of the CentOS6 libc, and how it works with the third-party libs.
+
+@smjert is working on this, with help from @theopolis
+
+We do not need to support _building_ osquery on CentOS6. But we do need to _run_ it there. The build/test process right now, is to build on Ubuntu, transfer the binaries to CentOS6 and then test. (As it happens, the build system does work on CentOS6)
+
+An example:
+* augeas needs `fnmatch`
+* The `fnmatch` in the older libc is buggy
+* So augeas has a `posix_fnmatch`
+* But autoconf won't detect it when it's running on a newer build system -- autoconf will make the best decision for the running environment, we need it to make the most portable decision.
+
+Discussion about shipping a libc? Conclusion is that this isn't something worth pursuing.
+
+There are some concerns about merging the toolchain work. As it stands, it probably breaks build and runtime on some older platforms. But, the current build uses the OS toolchain, which is also broken on those platforms. So merging is not worse.
+
+What if we limited the supported platforms? Would this let us ship faster? What would our messaging here be? Does anyone use CentOS6. We believe that we are as stable as in prior releases. But we understand more about third party libraries, and we think there are places we can improve for older versions of libc.
+
+More people to help update third-party libs would help. Stefano has done about seven libraries. We have about 40. Not all libraries are equal.
+
+#### How do we move forward?
+
+We **will** merge the custom toolchain work. This needs a small amount of doc work, and binaries somewhere. https://github.com/osquery/osquer
+
+_Then_ we will tackle the third-party dependencies one at a time, getting support for older libc into their configuration files. This work will be done a third-party lib at a time.
```

Comment:
Do folks want me to make tickets for each third party lib? I can automate it, it's easy, but it is 40 new tickets

Reply:
My take is that if we expect people to subdivide work, and do this gradually, I would say that we need them to be separate.
At the same time, we should though have a single issue which covers CentOS 6 support (or similar) that collects them all, with a check list for each of them.
Much like https://github.com/osquery/osquery/issues/5652.

Reply:
Works for me. https://github.com/osquery/osquery/issues/5745